lib/debug/session.rb

```diff
@@ -14,6 +14,7 @@
 require_relative 'breakpoint'
 require_relative 'thread_client'
 require_relative 'config'
+require_relative 'console'
```

Comment:
Could you move it to the beginning of `def self.console`?

Reply:
Done.